tests/services/test_auditlogs.py

```diff
@@ -0,0 +1,99 @@
+from py42.services.audit_logs import AuditLogsService
+
+
+class TestAuditLogService(object):
```

Comment:
For record keeping purposes, it'd be nice to have a test for that is only for `get_page()`... I know that's probably covered in the other tests, but it would still be better to have.

Reply:
added.